kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -405,16 +422,31 @@ def cargo_check_toolchain_version(
 
     project = project or Project.current()
     task = project.task(
-        f"cargoCheckVersion/{minimal_version}", CargoCheckToolchainVersionTask, group=CARGO_BUILD_SUPPORT_GROUP_NAME
+        f"cargoCheckVersion/{minimal_version}",
+        CargoCheckToolchainVersionTask,
+        group=CARGO_BUILD_SUPPORT_GROUP_NAME,
     )
     task.minimal_version = minimal_version
     return task
 
 
-def rustup_target_add(target: str, *, group: str | None = None, project: Project | None = None) -> RustupTargetAddTask:
+def rustup_target_add(
+    target: str, *, group: str | None = None, project: Project | None = None
+) -> RustupTargetAddTask:
     """Creates a task that installs a given target for Cargo"""
 
     project = project or Project.current()
     task = project.task(f"rustupTargetAdd/{target}", RustupTargetAddTask, group=group)
     task.target = target
     return task
+
+
+def cargo_generate_deb_package(*, package_name: str):
+    project = project or Project.current()
```

Comment:
`project` is not defined here, you need:

```suggestion
def cargo_generate_deb_package(*, project: Project | None = None, package_name: str):
    project = project or Project.current()
```